sycl/include/sycl/info/device_traits.def

```diff
@@ -190,9 +190,9 @@ __SYCL_PARAM_TRAITS_SPEC(device, usm_restricted_shared_allocations, bool,
                          PI_USM_CROSS_SHARED_SUPPORT)
 __SYCL_PARAM_TRAITS_SPEC(device, usm_system_allocations, bool,
                          PI_USM_SYSTEM_SHARED_SUPPORT)
+// To be dropped (no alternatives)
 __SYCL_PARAM_TRAITS_SPEC(device, image_max_array_size, size_t,
                          PI_DEVICE_INFO_IMAGE_MAX_ARRAY_SIZE)
-// To be dropped (no alternatives)
```

Comment:
This should still say to be dropped, but in favor of `get_backend_info`

Reply:
Thanks for the comments! And for this one, I moved the entry for opencl_c_version up into the section of "// To be dropped (has alternatives/not needed)", but didn't add comment about get_backend_info here as we already have that mentioned in device_traits_deprecated.def